src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,30 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  gap: 48px;
+  max-width: 944px;
+  margin: 0 auto;
+
+  @media(min-width: 488px) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @media(min-width: 768px) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media(min-width: 1024px) {
+    grid-template-columns: repeat(4, $card-width);
+  }
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $card-width;
+    height: 300px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+    margin-bottom: 48px;
```

Comment:
Gap between cards must be 48px in total. If you have gap: 48px and margin-botton: 48px than the gap is 96px.